### PR TITLE
AB#1006: updating DEVELOPMENT

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
- updated DEVELOPMENT read me to have docker tips and release steps
- deleted surplus docker readme

### CODE REVIEW

File: README-DEVELOPMENT.md [hunk 1]
```diff
@@ -164,6 +164,24 @@ eze test
 ```
 </details>
 
+## Release Steps
+
+For repo maintainers
+
+_requires pypi and dockerhub access_
```
Comment (mgonzales-rs) [inline README-DEVELOPMENT.md:171]: good spot, I was going to suggest checking if the dev was logged in.

File: README-DEVELOPMENT.md [hunk 2]
```diff
@@ -164,6 +164,24 @@ eze test
...
+
+- update version in ```eze-cli\eze\__init__.py```
+- update change log with any changes
```
Comment (mgonzales-rs) [inline README-DEVELOPMENT.md:174]: maybe adding:  make a list of the changes with format > `ab-###: <brief description>`

File: README-DEVELOPMENT.md [hunk 3]
```diff
@@ -164,6 +164,24 @@ eze test
...
+- deploy to pypi
+  ```bash
+  make release-pypi-test
+  make release-pypi
+  ```
+- deploy to dockerhub
+  ```bash
+  release-docker
```
Comment (mgonzales-rs) [inline README-DEVELOPMENT.md:182]: `make release-docker`

**Review by mgonzales-rs — APPROVED** [review]

File: README-DEVELOPMENT.md (hunk 1 above)
Comment (Anthony-Mckale) [reply README-DEVELOPMENT.md:171]: :)

File: README-DEVELOPMENT.md (hunk 2 above)
Comment (Anthony-Mckale) [reply README-DEVELOPMENT.md:174]: +1

File: README-DEVELOPMENT.md (hunk 3 above)
Comment (Anthony-Mckale) [reply README-DEVELOPMENT.md:182]: +1

**Pull request merged**